### ISSUE TITLE
viber complains for old version of PulseAudio server

### ISSUE THREAD

**Issue by deemytch** [i-rinat/apulse#114]
archlinux

```
% yay -Qs viber 
local/viber 12.0.0.7-1
```

```
$ apulse viber
PulseAudioService: failed to subscribe to context notifications
qml: CALL_WEB_VIEW_DEBUG: callWebEngineLoader active: false
# theres more crap about qml and qrc
...
Error: "PulseAudio server version is too old."

```

This error is shown when I try to play received video in the chat.
That's all I know.

**Comment by krackout** [comment]
I have the same problem. I tried to recompile `apulse` after changing `apulse/3rdparty/pulseaudio-headers/pulse/version.h` strings where pulse versions are mentioned. I changed them to the latest ones. `apulse/src/apulse-volume.c` had to be changed also. Yet I still had the same error.

Nevertheless, to my amazement, viber calls can be made without pulseaudio (or apulse of course)! Just start the app, you'll see the error in console, but you can make calls. Adjust audio levels, out and mic, using alsamixer. Also regarding received media files, videos, audio recordings etc, they open in external players, mpv in my case, so playback is fine for them also.

So in a nutshell, ALSA only viber: Calls ok, media files ok, no sound notifications.
